components/tidb_query_vec_executors/src/fast_hash_aggr_executor.rs

```diff
@@ -392,6 +413,35 @@ where
     Ok(())
 }
 
+fn extend_states_for_each_row<T>(
```

Comment:
I expect the function name to reflect the fact that it is only used for "group by constant". But I don't have a good idea now. What do you think?

Reply:
@sticnarf  something like `handle_scalar_group`?